### DISCUSSION TITLE
What's up with the SNPE provider?

### DISCUSSION THREAD

**Discussion by aurelienpre** [microsoft/onnxruntime#16214] [General] [answered]
Hi all,

I am currently using the SNPE Provider of ONNX Runtime to run inferences on Android devices with Qualcomm chips. The related doc used to be at https://onnxruntime.ai/docs/execution-providers/SNPE-ExecutionProvider.html but has recently been removed (now point to a "Page not found" error).
The SNPE Provider code is still there in `master` (https://github.com/microsoft/onnxruntime/tree/main/onnxruntime/core/providers/snpe).
What is the status is the SNPE Provider moving forward? Is this deprecated? Should I migrate my implementations to using the QNN Provider instead (https://onnxruntime.ai/docs/execution-providers/QNN-ExecutionProvider.html).
Thanks for your help clarifying this.
Cheers,
Aurelien

**Comment by HectorSVC** [comment, marked as answer]
That's right. We are deprecating SNPE EP.  Please use QNN instead.

**Reply by aurelienpre** [reply]
Thanks for your reply Hector, I will then move to QNN.

Would have been nice to get that announced rather than just silently nuking the doc IMO (if it was communicated, then I missed it). Any chance the SNPE Execution Provider doc could be restored with the deprecation mention, for current users ?
